.github/workflows/ci-build.yml

```
@@ -0,0 +1,20 @@
+name: ci-build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Build Docker images
+      run: |
+        cd base && ./build-all-build.sh
```

Probably a little cleaner to do:
```suggestion
    - name: Build Docker images
      working-directory: base
      run: ./build-all-build.sh
```

---

updated